makerdao/governance-portal-v2
# Bug/sc 860/on small mobile devices vote on executive

### Link to Shortcut ticket:
https://app.shortcut.com/dux-makerdao/story/860/on-small-mobile-devices-vote-on-executive-modal-is-too-big-for-the-screen

### What does this PR do?
Fixes issue with small screens and mobile dialog being too tall.

### Steps for testing:
1. Go to executive page
2. Simulate iPhone SE in browser
3. See that modal is now fully viewable (scroll for rest of content)

### Screenshots (if relevant):
<img width="343" alt="Screen Shot 2021-12-15 at 4 35 33 PM" src="https://user-images.githubusercontent.com/5225766/146216415-906fdf70-d054-45e1-b07a-cccffccad0af.png">

### Add a GIF:
![](https://media.giphy.com/media/vwjzWQxKL1mM4YKRzm/giphy.gif)
